src/universal/styles/providers.js

```diff
@@ -0,0 +1,10 @@
+// for provider UI components
+
+export const GITHUB_NAME = 'GitHub'
```

Comment:
let's put all these inside `universal/constEnums.d.ts` under an enum like `Provider`.
That way, the compiled code doesn't have to import anything, the references are just replaced by the values